### DISCUSSION TITLE
Importing data into table having LinkToAnotherRecord relation

### DISCUSSION THREAD

**Discussion by MGW-SumathyV** [nocodb/nocodb#3772] [Q&A]
Hi,

I imported data into TableB  (using csv) that has **Name** column having **LinkToAnotherRecord** relation with **TableA** (username column).
The **Name** column in TableB is not getting imported (**Name** column has  value consistent with TableA)  but other columns are imported.
How to resolve this issue ? It would be great help if its resolved

Thanks and Regards,
Sumathy Venugopal

**Comment by dstala** [comment]
@wingkwong do we support imports to a relational column (BT, HM, MM)?

**Reply by wingkwong** [reply]
Currently it's not supported.

**Reply by MGW-SumathyV** [reply]
Is there any planned release date for this feature?

**Comment by dstala** [comment]
@MGW-SumathyV - you can also connect with us over [Discord](https://discord.gg/5RgZmkW)
